chatbot.py

```diff
@@ -61,7 +67,10 @@
 while True:
     speech_type = input('Speech/Text: ')
     if speech_type.lower() != "speech":
-        translate = input("Type: ")
+        translate = input("Type: ").lower()
+        translate = translate.translate(table)
+        translate = spell.correction(translate)
+        # print(translate)
```

Comment:
what is the usecase of comment statement. can it notify user when wrong spell.

Reply:
Gargi, Any update on this?

Reply:
The commented statement was to check if the pyspellchecker object accurately corrects the misspelled words. Yes, we can uncomment it and show the user the corrected statement.

Reply:
Yes, can you remove this or put a comment regarding explaining how it works. #print looks like a debug statement.